R/TunerIrace.R

```diff
@@ -108,8 +108,9 @@ TunerIrace = R6Class("TunerIrace",
         stopf("%s is not supported. Use <TerminatorEvals> or <TerminatorRunTime> instead.", format(inst$terminator))
       }
       
+      browser()
       # Set resampling instances
-      ri = map(seq(pv$n_instances), function(n) {
+      ri = replicate(seq(pv$n_instances), {
```

Comment:
```suggestion
      ri = replicate(pv$n_instances, {
```
did the other even work?

Reply:
Thanks. No I pushed that by mistake...